physlr/physlr.py

```diff
@@ -2236,24 +2237,208 @@ def physlr_bed_to_path(self):
             f"Wrote {num_contigs} contigs in {num_scaffolds} scaffolds.",
             file=sys.stderr)
 
-    def physlr_path_to_fasta(self):
+    @staticmethod
+    def read_arcs_pair(filename):
         """
-        Produce sequences in FASTA format from paths.
-        Usage: physlr path-to-fasta FASTA PATH... >FASTA
+        Read ARCS pair tsv. Return a dictionary of pairs to orientation evidence.
+        """
+        pairs = {}
+        with open(filename, "r") as arks_pair_file:
+            for line in arks_pair_file:
+                columns = line.rstrip().split("\t")
+                (u, v, hh, ht, th, tt) = [int(column) if idx > 1 else column
+                                          for idx, column in enumerate(columns)]
+                pairs[(u, v)] = [hh, ht, th, tt]
+                pairs[(v, u)] = [hh, th, ht, tt]
+        print("Read ARCS pairs", file=sys.stderr)
+        return pairs
+
+    @staticmethod
+    def read_dist_est(filename, dist_type):
+        """
+        Read ARCS distance estimates. Return a dictionary of pairs to distance estimates.
+        """
+        dist = {}
+        with open(filename, "r") as dist_est_file:
+
+            dist_type_to_idx = {"min":2, "avg":3, "max":4}
+
+            if dist_type not in dist_type_to_idx:
+                print("invalid --dist-type parameters. Acceptable values are: min, avg, max",
+                      file=sys.stderr)
+                sys.exit(1)
+            idx = dist_type_to_idx[dist_type]
```

Comment:
I believe that none of this code is reliant on the open file? I'd just move that up out of the `with open` block for clarity

Reply:
done